internal/resource/common/policyaction/policy_action_client_struct.go

```diff
@@ -68,6 +77,15 @@ func ClientStruct(object types.Object) (*client.PolicyActionAggregation, error)
 		if err != nil {
 			return nil, err
 		}
+
+		attributeSources := fragmentPolicyAction.(types.Object).Attributes()["fragment_mapping"].(types.Object).Attributes()["attribute_sources"]
+		if attributeSources != nil {
```

Comment:
It would be more go-like if these checks used a second `ok` parameter to verify that "attribute_sources" is in the map, rather than the nil check here